pkg/network/ebpf/c/protocols/postgres/types.h

```diff
@@ -6,8 +6,11 @@
 // Maximum length of Postgres query to send to userspace.
 #define POSTGRES_BUFFER_SIZE 160
 
-// Maximum number of Postgres messages we can parse for a single packet.
-#define POSTGRES_MAX_MESSAGES 80
+// Represents the maximum number of tail calls we can use to process a single message.
+#define POSTGRES_MAX_TAIL_CALLS_FOR_MAX_MESSAGES 4
+
+// Represents the maximum number of messages we process in a single tail call.
+#define POSTGRES_MAX_MESSAGES_PER_TAIL_CALL 20
```

Comment:
how far are we from the limits?
if we have enough room, try to increase the number of messages in each tail call to the maximum we can, and reduce the number of tail calls.
Each tail calls brings additional "latency", so we should favor less tail calls when possible